.release-please-manifest.json

```diff
@@ -1,5 +1,6 @@
 {
   "core/typescript": "2.0.0",
+  "samples/packages/pulumi-typescript": "1.0.0",
```

Comment:
Don't think you need to include this as release tooling will take care of this.

Reply:
if you need to tell it the starting version apparently manually add here. don't know if we do. given a 1.0.0 is published but not tracked here, I added.